src/Controllers/Version4/AbstractObjectsController.php

```diff
@@ -268,6 +280,23 @@ protected function get_objects( $query_args ) {
 			$count_query = new \WP_Query();
 			$count_query->query( $query_args );
 			$total_posts = $count_query->found_posts;
+		} elseif ( $this->support_eager_loading() ) {
+			$posts = get_posts(
+				array(
+					'include'     => $post_ids,
+					'post_type'   => $this->post_type,
+					'post_status' => $query_args[ 'post_status' ] ?? 'any',
```

Comment:
Isn't `??` a PHP 7 only operator?

Reply:
yes, but with WC 3.9 we update PHP requirements to 7.x. I am hoping that with next version of WC REST API we can update PHP requirements to 7.x as well, but if not I will revert this to use `isset` instead.